client/package.json

```diff
@@ -20,7 +20,7 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "//": "The sed command here is a temporary workaround to fix an issue in graphql-codegen",
-    "generate": "cd ../server && cargo run --bin remote_server_cli -- export-graphql-schema && cd ../client && graphql-codegen --config codegen.yml && find . -type f -name 'operations.generated.ts' -exec perl -i -pe 's/graphql-request\\/dist\\/types\\.dom/graphql-request\\/src\\/types\\.dom/g' {} \\;",
+    "generate": "cd ../server && cargo run --bin remote_server_cli -- export-graphql-schema && cd ../client && graphql-codegen --config codegen.yml",
```

Comment:
no longer required, and results in branch differences to the generated files. This is now handled by an alias in tsconfig.